dbt_utils_integration_tests_howto/integration_tests__tests__jinja_helpers__test_slugify__sql.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/integration_tests/tests/jinja_helpers/test_slugify.sql b/integration_tests/tests/jinja_helpers/test_slugify.sql
+index 7d07ec4..a95c324 100644
+--- a/integration_tests/tests/jinja_helpers/test_slugify.sql
++++ b/integration_tests/tests/jinja_helpers/test_slugify.sql
+@@ -1,9 +1,19 @@
+ with comparisons as (
+-  select '{{ dbt_utils.slugify("!Hell0 world-hi") }}' as output, 'hell0_world_hi' as expected
+-  union all
+-  select '{{ dbt_utils.slugify("0Hell0 world-hi") }}' as output, '_0hell0_world_hi' as expected
++    SELECT
++        CASE WHEN x.id = 1 THEN '{{ dbt_utils.slugify("!Hell0 world-hi") }}'
++             WHEN x.id = 2 THEN '{{ dbt_utils.slugify("0Hell0 world-hi") }}'
++        END AS output,
++        CASE WHEN x.id = 1 THEN 'hell0_world_hi'
++             WHEN x.id = 2 THEN '_0hell0_world_hi'
++        END AS expected
++    FROM
++    (SELECT CALENDAR_DATE AS _day_
```

Comment:
Can you please explain, why we need these queries to get an empty row?

Reply:
teradata does not support anonymous unions, intersects and differences.
writing something like this

> select '!Hell0 world-hi' as output, 'hell0_world_hi' as expected
> 
> union all
> 
> select '0Hell0 world-hi' as output, '_0hell0_world_hi' as expected

will give you an error saying "select from union, intersect or minus requires a table"

Using calendar table is a way to work around this limitation, the table generates some numbers and I use them to assign the values to the rows. This is just test setup, it is not validation of the semantic of the test